README.md

```diff
@@ -160,6 +161,64 @@ export class AppComponent {
 ></highcharts-chart>
 ```
 
+### Angular Elements and useHTML
+
+First, install angular elements:
+
+```cli
+npm install @angular/elements --save
+```
+
+Include in main.ts file your element tag inside allowedTags and [element properties](https://angular.io/guide/elements#mapping) inside allowedAttributes:
+
+```ts
+if (Highcharts && Highcharts.AST) {
+  Highcharts.AST.allowedTags = [
+    ...Highcharts.AST.allowedTags,
```

Comment:
Instead of the spread, official documentation suggests using push- [see](https://api.highcharts.com/class-reference/Highcharts.AST).

Reply:
Change pushed to the [master](https://github.com/jonancastelo/highcharts-angular) branch on [jonancastelo/highcharts-angular](https://github.com/jonancastelo/highcharts-angular).